dispatcher.sh

```diff
@@ -661,6 +678,11 @@ process_opts()
                 TE_TESTER_SCRIPTS="${TE_TESTER_SCRIPTS} ${script_file}"
                 ;;
 
+            --finish-script=* )
+                process_script_opt $1 finish-script
+                TE_FINISH_SCRIPTS="${TE_FINISH_SCRIPTS} ${script_file}"
```

Comment:
I think bash array would be better here to avoid issue with spaces in path.
May be tester script should be fixed before the patch as well. Please. do.